src/cpp/CMakeLists.txt

```diff
@@ -456,13 +457,55 @@ elseif(NOT EPROSIMA_INSTALLER)
     target_link_libraries(${PROJECT_NAME} ${PRIVACY} fastcdr foonathan_memory
         ${CMAKE_THREAD_LIBS_INIT} ${CMAKE_DL_LIBS}
         ${TINYXML2_LIBRARY}
-        $<$<BOOL:${LINK_SSL}>:OpenSSL::SSL$<SEMICOLON>OpenSSL::Crypto>
+        $<$<BOOL:${LINK_SSL}>:OpenSSL::SSL$<SEMICOLON>OpenSSL::Crypto$<$<BOOL:${WIN32}>:$<SEMICOLON>crypt32.lib>>
```

Comment:
I think `$<$<BOOL:${WIN32}>:$<SEMICOLON>crypt32.lib>` is not necessary.

Reply:
It turns out that [FindOpenSSL module](https://raw.githubusercontent.com/Kitware/CMake/master/Modules/FindOpenSSL.cmake) doesn't include this dependency and must be workaround locally.